src/FluentMigrator.DotNet.Cli/Commands/ConnectionCommand.cs

```diff
@@ -25,17 +25,20 @@ namespace FluentMigrator.DotNet.Cli.Commands
     public class ConnectionCommand : MigrationCommand
     {
         [Option("-c|--connection <CONNECTION_STRING_OR_NAME>", Description = "The name of the connection string (falls back to machine name) or the connection string itself to the server and database you want to execute your migrations against.")]
-        public string ConnectionString { get; }
+        public string ConnectionString { get; set; }
```

Comment:
The setter isn't needed by the command line parser/processor library.

Reply:
In the official examples only boolean properties has only `getter`.

https://github.com/natemcmaster/CommandLineUtils/blob/master/docs/samples/subcommands/inheritance/Program.cs